api/buckets/service.go

```diff
@@ -564,6 +570,13 @@ func (s *Service) Root(ctx context.Context, req *pb.RootRequest) (*pb.RootReply,
 	if err != nil {
 		return nil, err
 	}
+
+	buckPath := path.New(buck.Path)
+	stat, err := s.IPFSClient.Object().Stat(ctx, buckPath)
```

Comment:
Uhm, I think this doesn't work correctly for encrypted buckets?
cc @sanderpick

Reply:
Hrm, yeah... moving to fast the AM. Doing this properly could lead to an explosion... maybe we should avoid returning total sizes after all. Thoughts?

Reply:
Yeah, it would take a lot of computation to resolve in the encrypted case at least on the fly.
A solution to avoid that is doing in `sumBytesPinned` a counting also per bucket, so to have that pre-calculated.
In short, instead of only adding size changes account/user-wise, also take advantage and do that bucket wise.

Not sure what to do now though... I liked this test. 
Maybe freeze PR and create an issue if the above makes sense? (not sure is correct, was first idea on my mind)

Reply:
yeah, if a hassle, no problem. just thought it was a freebie. removing.

Reply:
Sounds good. I like the idea of tracking bucket size as well as total buckets size. Let's track an issue for now.

Reply:
okay, removed. test is still possible because the cumulative size is still reported in the list api